examples/otel-wrapper/src/main/java/com/splunk/support/lambda/examples/LambdaConfiguration.java

```diff
@@ -50,7 +50,7 @@ private static void configureLogging() {
   private static void configureOpenTelemetry() {
 
     OpenTelemetrySdk.builder()
```

Comment:
shouldn't we switch to using auto-configuration here?

Reply:
It's just an example to demonstrate how hard it is to use lambda wrapper without splunk-provided sugar. So I'd leave it as it is (hardcoded conf example).